Cauldron/Assets/Scripts/UI/RecipeBookButton.cs

```diff
@@ -15,16 +16,63 @@ public class RecipeBookButton : MonoBehaviour, IPointerClickHandler, IPointerEnt
         private Vector3 initialScale;
         private RectTransform transf;
 
+        private Canvas canvas;
+        private GraphicRaycaster raycaster;
+        private RectTransform bgButtonTransf;
+        private float initialPosXButton;
+        private float offPosXButton;
+        public float openCloseAnimationTime = 0.4f;
+
         [Inject]
         private RecipeBook book;
 
         private void Start()
         {
             transf = GetComponent<RectTransform>();
             initialScale = transf.sizeDelta;
+
+            canvas = GetComponentInParent<Canvas>();
+            raycaster = GetComponentInParent<GraphicRaycaster>();
+            bgButtonTransf = canvas.gameObject.GetComponent<RectTransform>();
+
+            initialPosXButton = bgButtonTransf.anchoredPosition.x;
+            Debug.Log(initialPosXButton);
+            offPosXButton = initialPosXButton + 210;
+            Debug.Log(offPosXButton);
+
+            book.ChangeBookButtonLayer += ChangeLayer;
+            book.AvailableChange += ChangeBookAvailable;
+        }
+
+        private void OnEnable()
+        {
             //start flashing to attract attention
-                transf.DOSizeDelta((initialScale * sizeCoef), sizeSpeed).
-                    SetLoops(-1, LoopType.Yoyo);
+            transf.DOSizeDelta((initialScale * sizeCoef), sizeSpeed).
+                SetLoops(-1, LoopType.Yoyo);
+        }
+
+        private void ChangeLayer(string name, int order)
+        {
+            canvas.sortingLayerName = name;            
+            canvas.sortingOrder = order;
+        }
+
+        private void ChangeBookAvailable(bool isAvailable)
+        {
+            if(isAvailable)
+            {
+                raycaster.enabled = true;
+                bgButtonTransf.DOAnchorPosX(initialPosXButton, openCloseAnimationTime).
```

Comment:
Try tweenSequence for chaining tweens - you can look up the example in EndGameFX script